js_modules/dagit/packages/core/src/gantt/GanttStatusPanel.tsx

```diff
@@ -56,18 +52,10 @@ export const GanttStatusPanel: React.FC<GanttStatusPanelProps> = ({
           break;
         case IStepState.SUCCEEDED:
           succeeded.push(key);
-          break;
       }
     }
-
-    for (const node of graph) {
-      const name = node.name;
-      if (!metadata.steps[name]?.state) {
-        notExecuted.push(name);
-      }
-    }
```

Comment:
this seems to be the source of the problem - there are nodes in the graph that don't have a state but still shouldn't be included in the list